pkg/foundation/metrics/measure/measure.go

```diff
@@ -32,8 +32,8 @@ var (
 		"Number of connectors by type.",
 		[]string{"type"})
 	ProcessorsGauge = metrics.NewLabeledGauge("conduit_processors",
-		"Number of processors by type.",
```

Comment:
Well, it is by processor name, and since we will rename field `name` to `type` it will be by type at that point 😄 We have to remember to update this gauge when renaming `name` to `type`.